ctapipe/image/muon/intensity_fitter.py

```diff
@@ -241,7 +242,9 @@ def image_prediction_no_units(
     # The weight is the integral of the ring's radial gaussian profile inside the
     # ring's width
     delta = pixel_diameter_rad / 2
-    cdfs = norm.cdf(
+
+    # Define Gaussian cdf for numba usage
```

Comment:
This comment does not really give more information since the function is aptly called `gaussian_cdf`